python/google/protobuf/internal/type_checkers.py

```diff
@@ -57,7 +62,7 @@
 
 
 def TruncateToFourByteFloat(original):
-  return ctypes.c_float(original).value
+  return struct.unpack('<f', struct.pack('<f', original))[0]
```

Comment:
If able to import ctypes, there's no struct. This should be:

   
 if ctypes:
      return ctypes.c_float(converted_value).value
    else:
      return struct.unpack('<f', struct.pack('<f', converted_value))[0]

Reply:
It appears that `struct` is available even in the most recent versions of Python: https://docs.python.org/3/library/struct.html

Can't we just use `struct` all the time?